lib/CustomFields/pages/EditCustomFieldsSettings/EditCustomFieldsSettings.js

```diff
@@ -91,14 +101,43 @@ const EditCustomFieldsSettings = ({
     }
   };
 
+  const handleOptionDelete = fieldId => optionData => {
+    const customField = customFields.find(cf => cf.id === fieldId);
+    const isOptionSaved = customField.selectField.options.values.find(option => optionData.id === option.id);
+
+    if (isOptionSaved) {
```

Comment:
I'd format this block in this way:
```suggestion
    if (!isOptionSaved) {
      return;
    }
    
    setOptionsToDelete(current => {
      const newOptionsToDelete = { ...current };
      const fieldHasOptionsToDelete = fieldId in optionsToDelete;
      const option = {
        fieldName: customField.name,
        optionData,
      };
      newOptionsToDelete[fieldId] = fieldHasOptionsToDelete
        ? [...newOptionsToDelete[fieldId], option]
        : [option];
        
      return newOptionsToDelete;
    });
```

Reply:
I don't like using `return` to just stop function execution because it's not semantically correct.
I'd prefer having human readable code, which focuses on what is happening in the app more:
```js
if(condition) {
  openModa();
  enableButton();
}
```
than having more technically detailed code 
```js
if(!condition) {
  return; // essentially, the same as stopFunctionExecution(). not very behavior driven-ish
}

openModa();
enableButton();
```

But in this particular case, it doesn't have much impact, so it's not my strong preference.

Reply:
In my opinion, if a function is called as handleOptionDelete, which implies that it is performing some action and returning nothing, then I see no need for an empty return.